spec/rubocop/cop/style/negated_if_spec.rb

```diff
@@ -68,6 +68,14 @@
     expect(cop.offenses).to be_empty
   end
 
+  it 'accepts an if where the condition is doubly negated' do
+    inspect_source(cop,
+                   ['if !!condition',
+                    '  some_method',
+                    'end'])
+    expect(cop.offenses).to be_empty
```

Comment:
I think there should also be a test for modifier `if` usage (single-line `if`). Refer [this case](https://github.com/natalzia-paperless/rubocop/blob/master/spec/rubocop/cop/style/negated_if_spec.rb#L14).